charts/operator/templates/role.yaml

```diff
@@ -274,6 +274,18 @@ rules:
     - patch
     - update
     - watch
+- apiGroups:
```

Comment:
Can you add these to our operator repo in the redpanda controller file, if you havent already. We also discussed that we will continue to hit these missing crds continually and maintaining this is going to be difficult.

Reply:
ahh next PR, nm then, thank you.